README.md

```diff
@@ -25,21 +25,21 @@ see the [Installing section in the user guide](https://www.montepy.org/starting.
 ## User Documentation
 
 MontePy has a [sphinx website](https://www.montepy.org/). 
```

Comment:
has a [documentation website for working with mcnp in python].

This is probably better for SEO.